ODIN_II/regression_test/benchmark/suite/keyword_suite/task_list.conf

```diff
@@ -0,0 +1,2 @@
+regression_test/benchmark/task/keywords/*
```

Comment:
the suite doesnt need to be called  `keyword_suite` given its in the suite directory